packages/react/src/components/form/form-container/form-field-container.tsx

```diff
@@ -0,0 +1,130 @@
+import { FunctionComponent, PropsWithChildren, useMemo } from 'react';
+import styled from 'styled-components';
+import { useId } from '../../../hooks/use-id';
+import { ResolvedTheme } from '../../../themes/theme';
+import { useDeviceContext } from '../../device-context-provider/device-context-provider';
+import { InvalidField } from '../feedbacks/invalid-field';
+import { Label } from '../../label/label';
+import { TooltipProps } from '../../tooltip/tooltip';
+import { FormFieldContext, FormFieldControlProps } from '../form-field-context';
+import { getAriaDescribedBy, getAriaLabel, getAriaLabelledBy, getSlotIds } from '../utils';
+
+interface StyledDivProps {
+    theme: ResolvedTheme;
+    hasLabel: boolean;
+    hasHint: boolean;
+    valid: boolean;
+    noMargin?: boolean;
```

Comment:
Tout en transient props

```suggestion
    $hasLabel: boolean;
    $hasHint: boolean;
    $valid: boolean;
    $noMargin?: boolean;
```